pupil/public/javascripts/mtc-check.js

```diff
@@ -1,25 +1,24 @@
-/* global Touch DocumentTouch Event */
-
 (function (global) {
   'use strict'
-  var GOVUK = global.GOVUK || {}
-  var startTime
-  var pageInterval = 5  // seconds allowed per page
-  var preloadInterval = 2 // seconds for student preparation
-  var deadline
-
-  var registerInputs = []
-  var isTouch = 'ontouchstart' in window || navigator.msMaxTouchPoints || navigator.maxTouchPoints ||
+
+  const GOVUK = global.GOVUK || {}
+  const defaultQuestionTimeLimit = 5  // seconds allowed per page
+  const defaultPreloadInterval = 2 // seconds for student preparation
+  const isTouch = 'ontouchstart' in window || navigator.msMaxTouchPoints || navigator.maxTouchPoints ||
     (window.Touch && document instanceof Touch) || (window.DocumentTouch && document instanceof DocumentTouch)
+
+  let deadline
```

Comment:
`let` won't work in IE9
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let